contracts/satoshi-bridge/src/network.rs

```diff
@@ -191,8 +191,8 @@ impl Address {
                                 })?,
                             ))
                         }
-                        _ => continue,
-                    };
+                        _ => {}
```

Comment:
Interesting, why we even had continue

Reply:
A unified address can contain different types of addresses, but we are only interested in two specific ones, so we search for those. If I remember correctly, only one of the types we’re interested in can be present.